docs/topics_5/plugin-components.adoc

```diff
@@ -1,23 +1,23 @@
 // Module included in the following assemblies:
 // * docs/plugin-guide_5/master.adoc
 [id='plugin_components_{context}']
-= About {PluginNameTitle} Components
+= About {PluginNameTitle} components
 
 The following components are available in the {ProductShortName} perspective when using the {PluginName} to analyze projects.
 
-Issue Explorer:: This view allows you to explore the {ProductShortName} issues for projects that have been analyzed.
+Issue explorer:: This view allows you to explore the {ProductShortName} issues for projects that have been analyzed.
```

Comment:
I checked the UI. Issue Explorer and the rest of these components are still "title case", so they have to be reverted. The plugin UI has not been updated for a while. Same for Issue Details and MTA Report. They have to be reverted.

BTW, whenever you change UI elements, you should **check the screenshots** before making changes to the text.

Reply:
Done.